[2025-02-09T05:15:37.362+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-02-09T05:15:37.389+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_MusicStreamPipeline.MusicStream_load_songplays manual__2025-02-09T05:15:30.006956+00:00 [queued]>
[2025-02-09T05:15:37.514+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_MusicStreamPipeline.MusicStream_load_songplays manual__2025-02-09T05:15:30.006956+00:00 [queued]>
[2025-02-09T05:15:37.515+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2025-02-09T05:15:37.526+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): MusicStream_load_songplays> on 2025-02-09 05:15:30.006956+00:00
[2025-02-09T05:15:37.534+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1550) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-09T05:15:37.535+0000] {standard_task_runner.py:63} INFO - Started process 1552 to run task
[2025-02-09T05:15:37.534+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_MusicStreamPipeline', 'MusicStream_load_songplays', 'manual__2025-02-09T05:15:30.006956+00:00', '--job-id', '427', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmpvbammv0b']
[2025-02-09T05:15:37.536+0000] {standard_task_runner.py:91} INFO - Job 427: Subtask MusicStream_load_songplays
[2025-02-09T05:15:37.576+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_MusicStreamPipeline.MusicStream_load_songplays manual__2025-02-09T05:15:30.006956+00:00 [running]> on host e5529003cf96
[2025-02-09T05:15:37.648+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Le Huynh Thanh Duong' AIRFLOW_CTX_DAG_ID='etl_MusicStreamPipeline' AIRFLOW_CTX_TASK_ID='MusicStream_load_songplays' AIRFLOW_CTX_EXECUTION_DATE='2025-02-09T05:15:30.006956+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-09T05:15:30.006956+00:00'
[2025-02-09T05:15:37.649+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-02-09T05:15:37.726+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-02-09T05:15:37.727+0000] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/etls/load.py", line 75, in load_fact_songplays
    cur.execute(user_select, (row.userId,))
psycopg2.errors.InvalidTextRepresentation: invalid input syntax for type integer: "8.0"
LINE 1: SELECT user_key FROM dim_users WHERE user_id = '8.0' 
                                                       ^

[2025-02-09T05:15:37.736+0000] {taskinstance.py:1205} INFO - Marking task as FAILED. dag_id=etl_MusicStreamPipeline, task_id=MusicStream_load_songplays, execution_date=20250209T051530, start_date=20250209T051537, end_date=20250209T051537
[2025-02-09T05:15:37.744+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 427 for task MusicStream_load_songplays (invalid input syntax for type integer: "8.0"
LINE 1: SELECT user_key FROM dim_users WHERE user_id = '8.0' 
                                                       ^
; 1552)
[2025-02-09T05:15:37.791+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-02-09T05:15:37.805+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-09T05:15:37.808+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
